web/styles/popovers.css

```diff
@@ -1251,6 +1291,8 @@ ul {
     }
 
     .status_emoji {
+        align-self: flex-start;
+        flex-shrink: 0;
```

Comment:
This prevents the status emoji from shrinking when we have a long status message.